### DISCUSSION TITLE
Error: 'git' must be installed and in your PATH!

### DISCUSSION THREAD

**Discussion by jhnworks** [Homebrew/discussions#1982] [Linux]
Though having the latest git version
[ @ ~]$ which git
/usr/local/git/bin/git
[ @ ~]$ git --version
git version 2.7.0

and, running "brew upgrade" results in 
[@~]$ brew upgrade >> /tmp/brew.upgrade.txt
Error: Please update your system Git.
Minimum required version: 2.7.0
Your Git version: ...
Your Git executable:
Updating Homebrew...

make[1]: Leaving directory `/tmp/binutils-20210815-3186-9528mu/binutils-2.36.1'
make: *** [all] Error 2

[ @ ~]$ cat /etc/redhat-release
Red Hat Enterprise Linux Server release 7.9 (Maipo)

How does go about resolving this.

**Comment by carlocab** [comment]
Do
```bash
export HOMEBREW_DEVELOPER=1
export HOMEBREW_GIT_PATH=/usr/local/git/bin/git
```
As suggested by the `HOMEBREW_DEVELOPER` flag, this isn't really a fully supported use case, since we expect Git to be found in a standard system path.

**Reply by jhnworks** [reply]
Even after setting this - 
[ @  ~]$ brew --version
Homebrew 3.2.6-123-g0e63fb8
You must: brew install git
Homebrew/homebrew-core (no Git repository)

[ @ ~]$ brew doctor
Warning: Your Homebrew's prefix is not /home/linuxbrew/.linuxbrew.
Some of Homebrew's bottles (binary packages) can only be used with the default
prefix (/home/linuxbrew/.linuxbrew).
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
unsupported configuration.
